include/zenoh-pico/net/query.h

```diff
@@ -23,7 +25,7 @@
 typedef struct {
     _z_value_t _value;
     _z_keyexpr_t _key;
-    _z_zint_t _qid;
+    uint32_t _rid;
```

Comment:
`z_declare_resource` returns an `uint16_t`-size `id`.
Any reason for a wider range here?

Reply:
Here, rid stands for request id. Changing the name to avoid confusions